lib/core/src/Cardano/Wallet/DB/Sqlite/TH.hs

```diff
@@ -173,6 +173,8 @@ ProtocolParameters
     protocolParametersDecentralizationLevel Percentage      sql=decentralization_level
     protocolParametersDesiredNumberOfPools  Word16          sql=desired_pool_number
     protocolParametersMinimumUtxoValue      W.Coin          sql=minimum_utxo_value
+    protocolParametersPoolDeposit           W.Coin          sql=pool_deposit
```

Comment:
This is superfluous for cardano-wallet, we do never register stake pools and won't do in any near nor distant future.